### PR TITLE
UIREC-188 Allow user to save and create another piece

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://issues.folio.org/browse/UIREC-188
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by AndreiShumski2 — APPROVED** [review]

**Comment by NikitaSedyx** [comment]
title details should know anything about isCreateAnother mode, it's form responsibility

File: src/TitleDetails/AddPieceModal/AddPieceModalContainer.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-import omit from 'lodash/omit';
+import { omit, noop } from 'lodash';
 
 import {
   ORDER_FORMATS,
   PIECE_FORMAT_OPTIONS,
   PIECE_FORMAT,
 } from '@folio/stripes-acq-components';
 
+import { getPieceIdFromCheckInResponse } from '../../common/utils';
```
Comment (NikitaSedyx) [inline src/TitleDetails/AddPieceModal/AddPieceModalContainer.js:11]: is it really common?

**Comment by NikitaSedyx** [comment]
why can't you call onCheckin and onSubmit with isCreateAnother flag?

File: src/TitleDetails/AddPieceModal/AddPieceModalContainer.js (hunk 1 above)
Comment (usavkov-epam) [reply src/TitleDetails/AddPieceModal/AddPieceModalContainer.js]: I think not really) updated

**Comment by usavkov-epam** [comment]
> why can't you call onCheckin and onSubmit with isCreateAnother flag?

Yeah, it seems to make sense to do that. Thanks!

Updated

**Review by NikitaSedyx — APPROVED** [review]

**Review by AndreiShumski2 — APPROVED** [review]

**Pull request merged**